routes/drones.js

```diff
@@ -1,36 +1,72 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
 // require the Drone model here
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+const Drone = require("../models/Drone.model");
+
+router.get("/drones", async (req, res, next) => {
+  try {
+    const drones = await Drone.find();
+    res.render("drones/list", { drones });
+  } catch (error) {
+    console.log(error);
+  }
 });
 
-router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.get("/drones/create", async (req, res, next) => {
+  res.render("drones/create-form");
```

Comment:
This route only render a view so the `async` is no needed. You could see that inside the function you didn't use any `await`